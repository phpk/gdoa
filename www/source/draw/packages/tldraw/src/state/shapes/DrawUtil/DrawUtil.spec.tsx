import { Draw } from '..'

describe('Draw shape', () => {
  it('Creates a shape', () => {
    expect(Draw.create({ id: 'draw' })).toMatchSnapshot('draw')
  })
})
