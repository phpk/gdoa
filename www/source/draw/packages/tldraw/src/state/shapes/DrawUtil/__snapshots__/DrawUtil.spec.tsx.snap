// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draw shape Creates a shape: draw 1`] = `
Object {
  "childIndex": 1,
  "id": "draw",
  "isComplete": false,
  "name": "Draw",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "points": Array [],
  "rotation": 0,
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "draw",
}
`;
